_sass/salween-theme.scss

```diff
@@ -169,7 +169,7 @@ h1.navbar-header {
     padding: 50px 0 50px;
     color: $base-color-white;
     text-align: center;
-    background: #1988BD url("./assets/Images/BG-3.png") no-repeat left top;
+    background: #1988BD url("../images/BG-3.png") no-repeat left top;
```

Comment:
Ah OK, if this `scss` file is processed to `_site/assets/css/style.css`, then the relative path should work.

But it doesn't looks like that file is being processed --- I don't see its contents in the built site (`_site`):

```
13:49 john@X250:~/salween/salween.github.io(master)$ ag 1988BD
_sass/salween-theme.scss
172:    background: #1988BD url("../assets/Images/BG-3.png") no-repeat left top;
179:      border: 1px solid #1988BD;
```